orcid_hub/utils.py

```diff
@@ -132,7 +132,7 @@ def send_email(
     recipient,
     cc_email=None,
     sender=(app.config.get("APP_NAME"), app.config.get("MAIL_DEFAULT_SENDER")),
-    reply_to=None,
+    reply_to=app.config.get("MAIL_SUPPORT_ADDRESS"),
```

Comment:
This might cause the email marked as spam...

Reply:
I'm interested in why you think that? We've certainly never had that problem with other systems that use a noreply@ address from From: and a valid Reply-To:, and the config is quite common in e.g. mailing lists. But perhaps there's something peculiar about how this is being done that I don't understand?